### DISCUSSION TITLE
Pesquisa para desenvolvimento de app que sejam de utilidade pública.

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1557] [Discussão]
Nos ajude nesta Pesquisa: 
Um novo app, gratruito, open source, que irá ajudar muitas pessoas no Brasil!

Olá, sou Wilson, trabalho com desenvolvimento. Junto a amigos e colegas, para praticar algumas novas tecnologias, gostaríamos de sugestões de aplicativos que possam ajudar a sociedade de maneira geral, que sejam de utilidade pública, ou que de alguma maneira ajudem a preservar o meio ambiente...

Enfim, acredito que ficou clara a proposta, porém gostaríamos de levantar sugestões, pois pode surgir ideias bem legais e que será bem interessante desenvolver.

A ideia aqui é que seja um app, mas fique a vontade em dar sugestões que vão além....

O objetivo é que este app seja gratuito e open-source. 

Qual sua sugestão?

Responda na pesquisa abaixo:
https://forms.gle/gyHqgY3CUY3r9Tsw6

Por favor, compartilhe com mais pessoas para que assim possamos encontrar sugestões de soluções que realmente possam ajudar pessoas com algo que talvez nem tenhamos conhecimento.

**Comment by theandersonn** [comment]
Como a proposta é open-source o grupo que vocês representam tem uma org pública aqui no GitHub? Vai facilitar a troca de ideias, levantamento de requisitos etc...

Tem essa iniciativa aqui:
https://github.com/frontendbr/open-source

**Comment by wilsonneto-swe** [comment]
Na verdade não temos nada ainda, vamos começar com uma idéia -> escopo, ai criar os repos aqui, codar e divulgar.
